easybuild/framework/easyblock.py

```diff
@@ -1444,8 +1444,11 @@ def extensions_step(self, fetch=False):
         self.clean_up_fake_module(fake_mod_data)
 
     def package_step(self):
-        """Package software (e.g. into an RPM)."""
-        pass
+        """Prepare package software (e.g. into an RPM) with fpm."""
+
```

Comment:
We don't know typically, but everything is in place to start tracking licenses (see the `software_license` easyconfig parameter). We can assume the license is the most restrictive possible when `software_license` is set to `None`.